extensions/hibernate-reactive/deployment/pom.xml

```diff
@@ -133,7 +133,7 @@
             <plugin>
                 <artifactId>maven-surefire-plugin</artifactId>
                 <configuration>
-                    <skip>true</skip>
+                    <skip>false</skip>
```

Comment:
Probably added by mistake. Tests are skipped by default, that's normal: you need to pass `-Dstart-containers -Dtest-containers` to run them. I think that's mentioned in `CONTRIBUTING.md`.

Related: https://github.com/quarkusio/quarkus/issues/44124

Reply:
Yes, I need to revert this change

Reply:
> Probably added by mistake. Tests are skipped by default, that's normal: you need to pass `-Dstart-containers -Dtest-containers` to run them. I think that's mentioned in `CONTRIBUTING.md`.

This isn't true for every extension right? Because for some of them it enables test, but for example

```
./mvnw -f extensions/hibernate-orm clean install -Dstart-containers -Dtest-containers
```

Doesn't run hibernate-orm tests

Reply:
It's true for every extension that requires containers to run its tests.

`extension/hibernate-orm` uses h2 so doesn't require containers.

Not sure why tests do not run for you, they do for me with the command you gave.